app/src/common/shared/org/mozilla/vrbrowser/browser/engine/SessionStack.java

```diff
@@ -1149,100 +1147,95 @@ public void updateCursorAnchorInfo(@NonNull GeckoSession aSession, @NonNull Curs
 
     @Override
     public void onContentBlocked(@NonNull final GeckoSession session, @NonNull final ContentBlocking.BlockEvent event) {
-        if ((event.categories & ContentBlocking.AT_AD) != 0) {
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.AD) != 0) {
           Log.i(LOGTAG, "Blocking Ad: " + event.uri);
         }
 
-        if ((event.categories & ContentBlocking.AT_ANALYTIC) != 0) {
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.ANALYTIC) != 0) {
             Log.i(LOGTAG, "Blocking Analytic: " + event.uri);
         }
 
-        if ((event.categories & ContentBlocking.AT_CONTENT) != 0) {
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.CONTENT) != 0) {
             Log.i(LOGTAG, "Blocking Content: " + event.uri);
         }
 
-        if ((event.categories & ContentBlocking.AT_SOCIAL) != 0) {
+        if ((event.getAntiTrackingCategory() & ContentBlocking.AntiTracking.SOCIAL) != 0) {
             Log.i(LOGTAG, "Blocking Social: " + event.uri);
         }
     }
 
     // PromptDelegate
 
+    @Nullable
     @Override
-    public void onAlert(@NonNull GeckoSession session, String title, String msg, @NonNull AlertCallback callback) {
-        if (session == mCurrentSession) {
-            for (GeckoSession.PromptDelegate listener : mPromptListeners) {
-                listener.onAlert(session, title, msg, callback);
-            }
-        }
-    }
+    public GeckoResult<PromptResponse> onAlertPrompt(@NonNull GeckoSession geckoSession, @NonNull AlertPrompt alertPrompt) {
+        if (mPromptDelegate != null)
```

Comment:
nit: I prefer to always use `{}` with `if` statements. I've fixed too may bugs in the past where they were left off and then some one added a second line with out realizing the curly brackets were missing. I just grepped the code and see we have a lot of if statements with out `{}` so I will file a follow up to update the code.